change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added img aria role to <path/>",
```

Comment:
Update comment to include entry for SparklineChart

Reply:
Ok